core/templates/components/state-editor/state-interaction-editor/state-interaction-editor.component.ts

```diff
@@ -67,6 +67,7 @@ export class StateInteractionEditorComponent
   directiveSubscriptions = new Subscription();
   hasLoaded: boolean;
   interactionEditorIsShown: boolean;
+  showMe: boolean = true;
```

Comment:
Please use a more descriptive name for this variable.

Reply:
Done!